Gemfile

```diff
@@ -13,6 +13,8 @@ gem 'braque'
 gem 'haml'
 gem 'simple_form'
 
+gem 'albers', git: 'https://github.com/dylanfareed/albers'
```

Comment:
Distribution directly via GitHub for now. Didn't really want to pollute the RubyGems namespace.